### PR TITLE
vstart: update help with a few more commandline opts

### PR DESCRIPTION
**Pull request by theanalyst**
Adding opts like bluestore, memstore etc. to help and dev. docs

### CODE REVIEW

File: doc/dev/dev_cluster_deployement.rst [hunk 1]
```diff
@@ -73,6 +73,23 @@ Options
 
     Launch the osd/mds/mon/all the ceph binaries using valgrind with the specified tool and arguments.
 
+.. option:: --[{mon,osd,mds}]_num
```
Comment (tchaikov) [inline doc/dev/dev_cluster_deployement.rst]: `{mon,osd,mds}` is not optional here. so should drop `[]`.

File: doc/dev/dev_cluster_deployement.rst [hunk 2]
```diff
@@ -73,6 +73,23 @@ Options
...
+
+    Set the count of monitors, osds and mds
```
Comment (tchaikov) [inline doc/dev/dev_cluster_deployement.rst]: better off putting 

> Set the count of monitors, osds or mds

? i am referencing the notation of "osd/mds/mon/all" above, where "/" always means "or", i guess.

Comment (theanalyst) [inline doc/dev/dev_cluster_deployement.rst]: Sure makes sense

**Comment by theanalyst** [comment]
@tchaikov updated with your comments

**Comment by theanalyst** [comment]
@tchaikov updated again, Misunderstood your earlier comment :)

**Comment by theanalyst** [comment]
@tchaikov Fixed now

**Comment by tchaikov** [comment]
lgtm

**Pull request merged**